tests/scripts/pulpcore/test_orphans.sh

```diff
@@ -6,3 +6,40 @@
 expect_succ pulp orphan cleanup
 
 test "$(echo "${OUTPUT}" | jq -r '.state' )" = "completed"
+
+pulp debug has-plugin --name "file" || exit 3
+
+if pulp debug has-plugin --name "core" --max-version "3.13.0"
```

Comment:
```suggestion
if ! pulp debug has-plugin --name "core" --min-version "3.14.0"
```
We would handle 3.13.1 wrong...